spec/features/help_modal_spec.rb

```diff
@@ -0,0 +1,51 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Help modal' do
+  let(:user) { create(:user) }
+
+  context 'when authenticated' do
+    before do
+      sign_in user
+      allow(Settings).to receive(:allow_sdr_content_changes).and_return(true)
+    end
+
+    it 'opens modal, enters text, and submits query' do
+      visit '/'
+      click_link 'Help'
+      expect(page).to have_content 'Required fields'
+      # Email is pre-populated from user
+      expect(page).to have_field('What is your email address?*', with: user.email)
+      # Fills in remaining fields
+      fill_in 'What is your name?', with: 'User One'
+      fill_in 'What is your Stanford affiliation/department?', with: 'SUL'
+      select 'I want to ask a question', from: 'How can we help you?*'
+      fill_in 'Describe your issue, question, or what you would like to deposit', with: 'A question for the ages'
+      click_button 'Submit'
+      expect(page).to have_content '{"status":"success"}'
```

Comment:
This is returned JSON response but not sure how to test the new DOM elements that are created in the UI.

Reply:
Should just be `expect(page).to have_content 'You should receive a response from our team within 48 hours.'`

Reply:
I originally had `expect(page).to have_content 'You should receive a response from our team within 48 hours.'` in the test but the session Capybara only returns the `{"status": "success"}` and not the expected HTML.

Reply:
@jermnelson I wonder if you need to add a `js: true` like [here](https://github.com/sul-dlss/happy-heron/blob/main/spec/features/create_new_work_spec.rb#L6) to see this change work

Reply:
@mjgiarlo thanks! That was it!